### ISSUE TITLE
Error reported in CADENCE 17.4

### ISSUE THREAD

**Issue by strangerby** [juulsA/exportJson#1]
At the command line, enter exportjson and enter error when prompted。

exportjson
Exporting .json for variant: EUROPE
*Error* pcreReplace: argument #2 should be a string (type template = "ottxx") - t

**Comment by juulsA** [comment]
Is it possible for you to share the design? Unfortunetly debugging in the skill environment is not as easy as one might think ...

**Comment by strangerby** [comment]
[demo.zip](https://github.com/juulsA/exportJson/files/10731802/demo.zip)
This is my test project.

> 您可以分享设计吗？不幸的是，在技能环境中调试并不像人们想象的那么容易......

**Comment by juulsA** [comment]
Thank you for sharing! I can reproduce the error and look into it as soon as possible!

**Comment by strangerby** [comment]
> 谢谢你的分享！我可以重现错误并尽快调查！

Thank you.

**Comment by juulsA** [comment]
Check out my last commit and let me know, if it is working for you.
I could find the reason for the error and made a quick fix, which should solve your problem, but I'm not sure, if there is a little bit more to do ...

**Comment by strangerby** [comment]
> Check out my last commit and let me know, if it is working for you. I could find the reason for the error and made a quick fix, which should solve your problem, but I'm not sure, if there is a little bit more to do ...
I used the new version and still the same error. Here are the statements I added in allegro.ilinit。
![image](https://user-images.githubusercontent.com/45263752/218735810-99990ad6-eef6-43b4-8099-932d9cad4ad1.png)

**Comment by strangerby** [comment]
This is lastly error.

*WARNING* No variant.lst found, exporting entire design ... 
Loading skillExt.cxt 
*Error* pcreReplace: argument #2 should be a string (type template = "ottxx") - t

**Comment by strangerby** [comment]
> 

I deleted the previous file and now it works. Thank you

**Comment by juulsA** [comment]
In allegro you have to be careful: deleting a file does not affect the code execution immediately, but only after the next restart. If you want to see an effect directly, you need to reload the file by tying `load( filename )`.

**Comment by strangerby** [comment]
Found a new issue, I've successfully exported json, but conversion fails with InteractiveHtmlBom.
![image](https://user-images.githubusercontent.com/45263752/218747005-d7e6b8fc-775d-4b61-aa6a-d79d1392c9f4.png)
[test.zip](https://github.com/juulsA/exportJson/files/10732573/test.zip)

**Comment by juulsA** [comment]
Did you downloaded the newest version of the InteractiveHtmlBom?

I think this is a [bug](https://github.com/openscopeproject/InteractiveHtmlBom/issues/337#issuecomment-1368933089), which should be fixed now.

**Comment by strangerby** [comment]
> Did you downloaded the newest version of the InteractiveHtmlBom?
> 
> I think this is a [bug](https://github.com/openscopeproject/InteractiveHtmlBom/issues/337#issuecomment-1368933089), which should be fixed now.

Thanks. I used the last InteractiveHtmlBom,it is successed.
